examples/doge-example/project.json

```diff
@@ -4,26 +4,14 @@
   "projectType": "application",
   "targets": {
     "build": {
-      "executor": "@nrwl/node:build",
+      "executor": "@nrwl/node:package",
       "outputs": ["{options.outputPath}"],
       "options": {
         "outputPath": "dist/examples/doge-example",
-        "main": "examples/doge-example/src/main.ts",
+        "main": "examples/doge-example/src/index.ts",
         "tsConfig": "examples/doge-example/tsconfig.app.json",
-        "assets": ["examples/doge-example/src/assets"]
-      },
-      "configurations": {
-        "production": {
-          "optimization": true,
-          "extractLicenses": true,
-          "inspect": false,
-          "fileReplacements": [
-            {
-              "replace": "examples/doge-example/src/environments/environment.ts",
-              "with": "examples/doge-example/src/environments/environment.prod.ts"
-            }
-          ]
-        }
+        "packageJson": "examples/doge-example/package.json",
+        "assets": ["examples/doge-example/*.md"]
```

Comment:
I think I should remove this completely as we don't have any assets

Reply:
+